roles/sap_hana_rhv_hypervisor/tests/test.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: localhost
+  remote_user: root
```

Comment:
This is a security risk, I have never seen that root will be allowed remote access. 
To mitigate this parameters become: true and become_user: root should be added to all tasks where sudo is required. 
This is my point of view.